samples/java/appgengine-java11/README.md

```diff
@@ -0,0 +1,68 @@
+# Snapshot Debugger Examples for Google App Engine Standard Environment for Java 11
+
+NOTE: This file was copied
+[README](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/appengine-java11/README.md)
```

Comment:
I like the style of referring to the source that you have in the simple-server README better.

Reply:
Did you simply mean with a bit of path context in the link summary? Or simply referring to the directly itself? Assuming just the path context I've updated it, let me know if it's not what you had in mind.